Codeinwp/otter-blocks
# Fix popup padding on mobile

Closes #1546.


### Summary


Swap the padding from the header to the content modal. This way it will make space for the `overflow`.

### Screenshots 

Before:

![image](https://user-images.githubusercontent.com/17597852/224061989-dbaf7007-2811-4686-8965-09cf275571fd.png)


After:

![image](https://user-images.githubusercontent.com/17597852/224061774-1c96bfcc-1404-4b07-b2b7-73e303cff18f.png)


----

### Test instructions


1. Add popup block.
2. Add some content with a big Heading at the begging.
3. Check if it is ok on mobile



---- 

### Checklist before the final review
